pkg/controllers/disruption/controller.go

```diff
@@ -184,7 +185,8 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command)
 		methodLabel:            m.Type(),
 		consolidationTypeLabel: m.ConsolidationType(),
 	}).Inc()
-	logging.FromContext(ctx).Infof("disrupting via %s %s", m.Type(), cmd)
+	uid := uuid.NewUUID()
```

Comment:
Should we just inject this into the logger generally at the start of the loop and then just pass it through any command execution that we get. That also means that you get any associative logging that happens in scheduling for free

Reply:
If you pass it through the context, you don't get it in the orchestration queue. So you could, but it would just be redundant anyway, since this is the only log in this function.